digital-twin/Samples/device/JdkSample/readme.md

```diff
@@ -0,0 +1,51 @@
+# Azure IoT Digital Twins Device SDK Sample
+
+**PREVIEW - WILL LIKELY HAVE BREAKING CHANGES**
+
+This project contains a sample implementation of a simulated environmental sensor. It shows how to:
+
+  * Implement the environmental sensor interface
+  * Create an interfaceInstance for this interface
+  * Use the digital twin device client to register this interfaceInstance and interact with the Digital Twins services.
+  * How to respond to command invocations
+  * How to respond to property updates
+  
+For Android, please see [here](../AndroidSample)
+## How to run the sample
+
+### Setup environment
+Prepare your platform following the instructions [here][devbox-setup] to install Java and Maven.
+If you want to run these samples in Intellij, follow [these][intellij-setup] instructions to prepare your Intellij settings
+
+### Sample Arguments
+
+In order to run this sample, you must set environment variables for:
+- "DIGITAL_TWIN_DEVICE_CONNECTION_STRING" : Your IoT Hub device's connection string
+
+### Other Prerequisites
+In order to run this sample, you will need an IoT Hub. You will also need at least one device registered in this hub so that the sample can register as that device
+* [Setup Your IoT Hub][lnk-setup-iot-hub]
+
+### Run the sample
+
+From the base digital-twin folder in this repo, run the following command:
+
+```sh
+mvn clean install -DskipTests
+```
```

Comment:
do you need -DskipTests for sample? Can you remove?

Reply:
Removed